wdfn-server/waterdata/views.py

```diff
@@ -23,6 +23,22 @@
 
 NWIS = NwisWebServices(app.config['SERVER_SERVICE_ROOT'], app.config['SITE_SERVICE_CATALOG_ROOT'])
 
+
+def has_feedback_link(request_endpoint):
```

Comment:
I am a little surprised this works. I would think you just call request.get('endpoint') or something like that. How does it get request_endpoint?

Reply:
lol. I knew I should have you review it. I pass 'request.endpoint' in, in the 'if' in the template.

Reply:
I'm okay with not using a context processor too. It's just something to consider for the future. Maybe just revert?